src/s1ap/handlers/attach_icsreq.c

```diff
@@ -49,7 +49,7 @@ get_icsreq_protoie_value(struct proto_IE *value, struct init_ctx_req_Q_msg *g_ic
 	ieCnt++;
 
 	/* E-RABToBeSetupItemCtxtSUReq start */
-	ERABSetup *e_rab = &(value->data[ieCnt].val.E_RABToBeSetupItemCtxtSUReq);
+	erab_setup_item *e_rab = &(value->data[ieCnt].val.erab_to_be_setup_item_ctxt_su_req);
 	/* TODO: Remove hardcoded values. */
 	e_rab->e_RAB_ID = 1;
 	e_rab->e_RAB_QoS_Params.qci = 9;
```

Comment:
Don't we get this from Mme-app ? What QCI to fill.

Reply:
This is the legacy code, that we retained from openmme. We don't get it from MME as of now.